airflow/macros/__init__.py

```diff
@@ -91,4 +92,4 @@ def _integrate_plugins():
     from airflow.plugins_manager import macros_modules
     for macros_module in macros_modules:
         sys.modules[macros_module.__name__] = macros_module
-        globals()[macros_module._name] = macros_module
+        globals()[macros_module.__name__.split('.')[-1]] = macros_module
```

Comment:
The `_name` attribute will be set during the [make_module](https://github.com/apache/airflow/blob/88989200a66291580088188f06a6db503ac823e2/airflow/plugins_manager.py#L159).

So related to that it needs to be:
```suggestion
        globals()[macros_module.__name__.lower().split('.')[-1]] = macros_module
```

But I am wondering why it is "private" when it gets only accessed from [inits](https://github.com/apache/airflow/search?utf8=%E2%9C%93&q=._name&type=) outside of its class. 
I suggest to ignore it via `pylint: disable=message`, because if you want to rename the `_name` attribute it will affect all other inits as well.

Reply:
It is used only in the following five [inits](https://github.com/apache/airflow/search?q=_integrate_plugins&unscoped_q=_integrate_plugins) in order to integrate plugins:

- [macros](https://github.com/apache/airflow/blob/88989200a66291580088188f06a6db503ac823e2/airflow/macros/__init__.py)
- [operators](https://github.com/apache/airflow/blob/5c7e4e4e9c2014a78830a81162532c0f2eecd6e4/airflow/operators/__init__.py)
- [sensors](https://github.com/apache/airflow/blob/5c7e4e4e9c2014a78830a81162532c0f2eecd6e4/airflow/sensors/__init__.py)
- [hooks](https://github.com/apache/airflow/blob/88989200a66291580088188f06a6db503ac823e2/airflow/hooks/__init__.py)
- [executors](https://github.com/apache/airflow/blob/88989200a66291580088188f06a6db503ac823e2/airflow/executors/__init__.py)

We could change ```_name``` to ```name```

Reply:
Yes, we could do that, but I think `_` does not ever mean it is "private". I have seen it in some places like in the gcp classes that are accessing those `_` variables. Because some are using it to mark a variable as "internal" which then would be totally okay to access from outside of its class.

Reply:
@BasPH What do you think since you introduced pylint?

Reply:
I don't see any problem naming it `name`, the name won't be edited and is fine to "expose".